src/assertions/CommonAssertions.ts

```diff
@@ -21,7 +21,7 @@ import { ZERO } from '../constants';
 
 export class CommonAssertions {
   public greaterThanZero(quantity: BigNumber, errorMessage: string) {
-    if (quantity.lte(ZERO)) {
+    if (!quantity || quantity.lte(ZERO)) {
```

Comment:
Why are we adding this check?

Reply:
Would we need to do the same for all the other assertions?

Reply:
i think we should for anything that requires it so that it doesnt throw an undefined error instead of the actual assertion error

Reply:
an alternative is to turn on `--strictNullChecks` and see what happens (will probably throw errors in specs or stuff). i think that checks to make sure variables cannot be null or undefined anywhere that they shouldnt be

Reply:
Wouldn't this be caught be Typescript on line 23?

Reply:
I'm assuming it would error with `Expected type BigNumber but got undefined`

Reply:
@inje no cuz we dont have strictNullChecks turned on in our config i think...we can try turning it on but it may cause some other unseen issues from other assumptions

Reply:
https://www.typescriptlang.org/docs/handbook/release-notes/typescript-2-0.html